cfme/tests/cloud_infra_common/test_policy_simulation.py

```diff
@@ -22,11 +22,18 @@
 
 
 @pytest.mark.parametrize("navigation", ["provider_vms", "all_vms", "vm_summary"])
+@pytest.mark.meta(blockers=[BZ(1717483, forced_streams=["5.11"])])
 @pytest.mark.uncollectif(
-    lambda navigation: navigation == "provider_vms" and (BZ(1686617).blocks or BZ(1686619).blocks),
+    lambda navigation: (
+        navigation == "provider_vms" and
+        (BZ(1686617, forced_streams=["5.10"]).blocks or
+         BZ(1686619, forced_streams=["5.10"]).blocks or
```

Comment:
Some questions here.

I don't see any clones of these two BZs for 5.10, do you think the fix will be backported? 

For 5.11 appliance forced_streams make no difference
![image](https://user-images.githubusercontent.com/42433123/59108173-9f959680-893a-11e9-8202-dce56fb95a17.png)

Reply:
@lina-nikiforova I don't think the BZs will be backported, which is why I've included the forced streams here. Regarding `1717539`, this required the `forced_streams` at the time I wrote it, I will go ahead and remove it though.

Reply:
@john-dupuy in that case, why not to uncollect for the 5.10 stream instead of checking for the conditions that will always be True?

Reply:
@lina-nikiforova That is because of the first condition, we still want to run this test case if `navigation != "provider_vms"`. But you're right, it doesn't make sense to have the condition that always returns true, I will put in a check for the 5.10 stream.